dango/genesis/src/lib.rs

```diff
@@ -21,11 +23,36 @@ pub type GenesisUsers = BTreeMap<Username, GenesisUser>;
 
 pub type Addresses = BTreeMap<Username, Addr>;
 
+const GUARDIANS_ADDRESSES: [&str; 19] = [
+    "WJO1p2w/c5ZFZIiFvczAbNcKPNM=",
+    "/2y5Ulib3oYsJe9DkhMvudSkIVc=",
+    "EU3oRgGTvfOi/PgfhqCXZfR2L9E=",
+    "EHoAhrMtegl3kmogUTHYcx05y+s=",
+    "jIKy/YL67ScR1Zrw8kmdFucm9rI=",
+    "EbOXVsBCRBvm2GULabVOvnFeI0M=",
+    "VM5bTTSPt0uVjolm4uw9vUlYp80=",
+    "FefK8HxOPcjnxGn5LIzYj7gAWiA=",
+    "dKO/kTlT1pUmDYi8GqJaTu42PvA=",
+    "AArAB2cns1++otrCj+5cyw/qdo4=",
+    "r0XO0Ta52eJJA0ZK6In1yKcj/BQ=",
+    "+TEkt8c4hDy7iehkyGLDjN3Mz5U=",
+    "0sw3pNwDao0jK0j2LN1HMUEvSJA=",
+    "2nmPaJajMx9ktIwS0dV/2cvnCBE=",
+    "caob4dNsr+OGeRD5nAnjR4mcGcM=",
+    "gZK25zh8zXaCd8F9qxt6UCfAs88=",
+    "F44hrS53rgZxFUnPux+cep2Alug=",
+    "XhSH81UV0CqSdTUEqNdUcbn0nts=",
+    "b768iY9APkdz6V/rFegMmpnINI0=",
+];
+
+const GUARDIAN_INDEX: u32 = 4;
```

Comment:
also i feel this should be in `examples/build_genesis.rs` instead of here. there's a chance the guardian set changes when we launch so it shouldn't be a constant in the library.